modules/vmseries/variables.tf

```diff
@@ -145,7 +169,7 @@ variable "identity_ids" {
 }
 
 variable "metrics_retention_in_days" {
-  description = "Specifies the retention period in days. Possible values are 0, 30, 60, 90, 120, 180, 270, 365, 550 or 730. Defaults to 90. A special value 0 disables creation of Application Insights altogether."
+  description = "Specifies the retention period in days. Possible values are 0, 30, 60, 90, 120, 180, 270, 365, 550 or 730. Defaults to 30 days."
```

Comment:
You say `defaults to 30` but the default is `null`. I know that you probably meant the Azure defaults, but in this case I would suggest leave a comment like `defaults to Azure default value`. Azure default tend to change w/o notice.

Maybe a link do documentation would also help. 

And please use multiline comment in this case (`EOF` style)